nmtwrapper.go

```diff
@@ -0,0 +1,142 @@
+package rsmt2d
+
+// The contents of this file have been adapted from the source file available at https://github.com/celestiaorg/celestia-app/blob/main/pkg/wrapper/nmt_wrapper.go,
```

Comment:
[question][not blocking] should we use a permalink in case this file moves? I don't have a strong preference

Reply:
Makes sense to me.  Revised the link https://github.com/celestiaorg/rsmt2d/pull/242/commits/a364d791b546ce08e8f9da5fa926bcd53a0e8f33.